ngx-fudis/projects/dev/src/app/components/formExamples.component.ts

```diff
@@ -63,10 +65,11 @@ export class AppFormExampleComponent implements OnInit {
 			null,
 			FudisValidators.required(this._translocoService.selectTranslateObject('form_errors.required'))
 		),
-		textInput: new FormControl<string | null | number>(null, [
-			FudisValidators.required(this._translocoService.selectTranslateObject('form_errors.required')),
-			FudisValidators.minLength(5, this._translocoService.selectTranslateObject('form_errors.notEnoughCharacters')),
-		]),
+		textInput: new FormControl<string | null | number>(null),
+		// textInput: new FormControl<string | null | number>(null, [
```

Comment:
Tätä sandboxia vois siistiä jos nää on tarkoitus poistaa. Samalla vois myös muokata templassa olevan tooltipin tekstin, koska se on nyt harhaanjohtava, koska sillä ei enää oo näitä validaattoreita.